### DISCUSSION TITLE
How to unalias a default letter?

### DISCUSSION THREAD

**Discussion by gaoqiangks** [kylechui/nvim-surround#249] [Q&A] [answered]
For example, the letter "b" is aliased to ")". However, I want the letter "b" for other usage.  The surrounds table doesn't cover the default aliases table, which I think aliases table are load after the surrounds table. So, is there any way to achieve this?

**Comment by kylechui** [comment, marked as answer]
Default settings can be disabled by setting the corresponding value to `false`.
